wherehows-web/app/routes/application.js

```diff
@@ -40,7 +46,14 @@ export default Route.extend(ApplicationRouteMixin, {
    * @override
    */
   async model() {
-    const isInternal = await get(this, 'configurator.getConfig')('isInternal');
+    const getConfig = get(this, 'configurator.getConfig');
+
+    const [isInternal, showStagingBanner, showStaleSearchBanner] = await Promise.all([
```

Comment:
Please see my comments below, the service can hold a reference to the configurator service to request these properties

Reply:
I'd rather not have a service import services, otherwise it starts to weave a web we don't want instead of having independent/separate components